src/main/java/org/veupathdb/service/sequence/model/ReferenceSequenceSpec.java

```diff
@@ -0,0 +1,78 @@
+package org.veupathdb.service.sequence.model;
+
+import com.fasterxml.jackson.annotation.JsonGetter;
+import com.fasterxml.jackson.annotation.JsonSetter;
+
+public class ReferenceSequenceSpec {
+  private String name;
+
+  @JsonGetter
```

Comment:
What benefit do these provide?  If serializing/deserializing JSON, I think Jackson will use all the getters by default; if you want it to skip, use @JsonIgore

Reply:
I don't know, didn't try without them! Good point probably.